hazelcast/src/main/java/com/hazelcast/map/impl/MapService.java

```diff
@@ -199,9 +205,9 @@ public void rollbackTransaction(UUID transactionId) {
         transactionalService.rollbackTransaction(transactionId);
     }
 
-    @Override
     public Map<String, LocalMapStats> getStats() {
-        return statisticsAwareService.getStats();
+        LocalMapStatsProvider localMapStatsProvider = mapServiceContext.getLocalMapStatsProvider();
```

Comment:
We don't use the `statisticsAwareService` anymore.

Reply:
Added back for now. (Tried to remove `StatisticsAwareService` entirely, but it can be done in 4.1 if MC doesn't depend on it anymore).